### DISCUSSION TITLE
How to have it press enter

### DISCUSSION THREAD

**Discussion by kellentow** [KarstSkarn/KasTAS#16] [Q&A] [answered]
I'm not sure if Kas can press enter but i just want to check.

**Comment by KarstSkarn** [comment, marked as answer]
Yes it can! You just need to change the assignation of keys in the _keydata_ file! Technically it supports all the keys listed here:

[https://learn.microsoft.com/en-us/dotnet/api/system.windows.forms.keys?view=windowsdesktop-7.0](https://learn.microsoft.com/en-us/dotnet/api/system.windows.forms.keys?view=windowsdesktop-7.0)

Being the name of the key that you must introduce into the config the left column of that table.

**Reply by kellentow** [reply]
Thanks!
